repo/blob/azure/azure_storage.go

```diff
@@ -232,6 +244,7 @@ func (az *azStorage) putBlob(ctx context.Context, b blob.ID, data blob.Bytes, op
 	}
 
 	if opts.HasRetentionOptions() {
+		// kopia delete marker blob can be Unlocked rather than Compliance
```

Comment:
Not sure this comment is relevant. Whether a blob is locked or not depends on how the user configured it, that is, the option specified in the configuration should be respected.

Reply:
```suggestion
		// kopia delete marker blob must be "Unlocked", so it cannot be overridden to "Locked" here.
```